src/PrestaShopBundle/Form/Admin/Type/TextWithRecommendedLengthType.php

```diff
@@ -34,7 +34,8 @@
 /**
  * Is used to add field with recommended input length counter to the form.
  *
- * Requires Javasript text-with-recommended-lenght-counter.js to work properly.
+ * Requires Javasript text-with-recommended-length-counter.js to work properly.
```

Comment:
```suggestion
 * Requires Javascript text-with-recommended-length-counter.js to work properly.
```

Reply:
Maybe we should link to the components documentation instead 🤔

Reply:
i was not sure if we should cross link between code & devdocs and devdocs & code ? but if it's not an issue, yes i can submit a new PR when the devdocs PR about this component is validated

Reply:
@thomasnares In that case you can write that it requires js component, not mention the file 👍🏻👍🏻👍🏻